Makefile.core.mk

```
@@ -20,7 +20,7 @@ OLD_VARS := $(.VARIABLES)
 -include Makefile.vendor.mk
 
 VERSION ?= 0.2.0
-MINOR_VERSION := $(shell v='$(VERSION)'; echo "$${v%.*}")
+MINOR_VERSION := $(shell echo "$${VERSION}" | cut -f1,2 -d'.')
```

```suggestion
MINOR_VERSION := $(shell echo "${VERSION}" | cut -f1,2 -d'.')
```

This will make it work

---

Thank you